core/commands/bitswap.go

```diff
@@ -6,17 +6,18 @@ import (
 	"io"
 
 	cmds "github.com/ipfs/go-ipfs/commands"
+	e "github.com/ipfs/go-ipfs/core/commands/e"
```

Comment:
@keks yeah your right, that slipped my mind, just "e" is a bit too terse though in my opinion.  Maybe "eror", but I am nitpicking now.